src/main/java/com/nvidia/spark/rapids/jni/GpuTimeZoneDB.java

```diff
@@ -273,15 +349,24 @@ private HostColumnVector getHostFixedTransitions() {
     }
   }
 
-  private Map<String, Integer> getZoneIDMap() {
+  public Map<String, Integer> getZoneIDMap() {
```

Comment:
Instead of making this public should we just add in the API that we really want?

```
int getIndexFor(zoneId ZoneId) {
...
```

Reply:
Done. Added comments.